Source: https://github.com/logrotate/logrotate/pull/24
Status: https://github.com/logrotate/logrotate/pull/24
Reason: enhancement

From 6b55ae74cec161a2b6508b485cf424720045f082 Mon Sep 17 00:00:00 2001
From: Simon Arlott <sa.me.uk>
Date: Tue, 15 Mar 2016 12:47:48 +0000
Subject: [PATCH] Destination already exists message should be output at error
 level

If an error like this occurs:
  destination /var/log/apport.log-201603 already exists, skipping rotation

There's no way to tell why logrotate has exited with an error. Raise the
log level of this message to ERROR so that the cause of the error is
available.
---
 logrotate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/logrotate.c b/logrotate.c
index 2bd2f44..4266b81 100644
--- a/logrotate.c
+++ b/logrotate.c
@@ -1561,7 +1561,7 @@ int prerotateSingleLog(struct logInfo *log, int logNum, struct logState *state,
 			message(MESS_FATAL, "could not allocate destFile memory\n");
 		}
 		if (!stat(destFile, &fst_buf)) {
-			message(MESS_DEBUG,
+			message(MESS_ERROR,
 					"destination %s already exists, skipping rotation\n",
 					rotNames->firstRotated);
 			hasErrors = 1;
